src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/Presentation/index.tsx

```diff
@@ -1,60 +1,517 @@
 import React from 'react';
 import { Controller } from 'react-hook-form';
-import { yupResolver } from '@hookform/resolvers/yup';
-import { DatePicker, FormGroup, Label } from '@trussworks/react-uswds';
+import { useTranslation } from 'react-i18next';
+import {
+  Fieldset,
+  FormGroup,
+  Grid,
+  Label,
+  TextInput
+} from '@trussworks/react-uswds';
+import {
+  SystemIntakeGRBReviewType,
+  UpdateSystemIntakeGRBReviewAsyncPresentationMutationVariables,
+  UpdateSystemIntakeGRBReviewStandardPresentationMutationVariables,
+  useUpdateSystemIntakeGRBReviewAsyncPresentationMutation,
+  useUpdateSystemIntakeGRBReviewStandardPresentationMutation
+} from 'gql/generated/graphql';
 
+import Alert from 'components/Alert';
+import DatePickerFormatted from 'components/DatePickerFormatted';
 import { EasiFormProvider, useEasiForm } from 'components/EasiForm';
+import FieldErrorMsg from 'components/FieldErrorMsg';
+import FileInput from 'components/FileInput';
+import HelpText from 'components/HelpText';
+import { TabPanel, Tabs } from 'components/Tabs';
 import { GRBReviewFormStepProps } from 'types/grbReview';
-import { GrbPresentationSchema } from 'validations/grbReviewSchema';
+import { fileToBase64File } from 'utils/downloadFile';
 
+import SendPresentationReminder from '../../SendPresentationReminder';
 import GRBReviewFormStepWrapper, {
   GRBReviewFormStepSubmit
 } from '../GRBReviewFormStepWrapper';
 
-type PresentationFields = {
-  grbDate: string;
-};
+type StandardPresentationFields =
+  UpdateSystemIntakeGRBReviewStandardPresentationMutationVariables;
+
+type AsyncPresentationFields =
+  UpdateSystemIntakeGRBReviewAsyncPresentationMutationVariables;
```

Comment:
Can you update these types to reflect the actual fields in the form? I think it gets a little confusing if our field values type is reflecting data that will never be there.

So for example the standard form only has `grbDate` and `presentationDeckFileData` fields, but the fields type shows a lot of additional fields from the mutation input (`systemIntakeID`, `recordingLink`, `transcriptFileData`, etc) that aren't part of the form.